kratos/includes/element.h

```diff
@@ -63,7 +63,8 @@ class Element : public GeometricalObject, public Flags
     ///@name Type Definitions
     ///@{
     /// Pointer definition of Element
-    KRATOS_CLASS_POINTER_DEFINITION(Element);
+    typedef Kratos::intrusive_ptr<Element> Pointer;
+    typedef Kratos::intrusive_weak_ptr<Element> WeakPointer;
```

Comment:
why not `KRATOS_CLASS_INTRUSIVE_POINTER_DEFINITION`?

Reply:
Indeed